nss-glamorous-chimpanzees/yak-boilerplateReset
# Pzc register

Related to Register new user ticket

```
git fetch --all
git checkout pzc-register
```

To test: 
1. On login page, click register
2. See form to create new user
3. Submit new information (checks email/user against DB, make sure this is unique)
4. Redirects to login, includes new username/password in login fields

5. On register, submit existing information. 
6. Warning below 

Still needs: 
1. Link on step 6 warning to click to go back to login
2. Initial text in login page has been removed, and now there's no indication of what those fields are. Need to add example text or labels. 

This touches: 

Login.js
Register.js
db.json
App.js
